omero/sysadmins/version-requirements.txt

```diff
@@ -0,0 +1,1883 @@
+********************
+Version requirements
+********************
+
+Proposal for revised minimum version requirements
+-------------------------------------------------
+
+Minimum software versions
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+This is a proposal to the wider community to solicit their
```

Comment:
Certainly happy to reword.  This section was intended to simply contain proposed changes along with a brief rationale, for people to read and comment on.  I wasn't thinking it would be retained for the 5.1 release; after we've reached a decision upon the minimum requirements, we can update the minimum requirements and delete this entirely?